src/openforms/submissions/migrations/0061_backfill_submissionfileattachment_variable.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.15 on 2022-10-11 12:51
+
+from django.db import migrations
+from django.db.models import Count
+
+
+def backfill_attachment_submission_variables_from_form_key(apps, schema_editor):
+    SubmissionStep = apps.get_model("submissions", "SubmissionStep")
+    SubmissionValueVariable = apps.get_model("submissions", "SubmissionValueVariable")
+
+    qs = SubmissionStep.objects.annotate(Count("attachments")).filter(
+        attachments__count__gt=0
+    )
+    qs = qs.only("id").prefetch_related("attachments")
+
+    for submission_step in qs.iterator():
```

Comment:
`qs.iterator()` discards the prefetch cache, so I'd either swap it around an loop over the attachments + do a `select_related` on the submission step, or drop the `prefetch_related` call.